client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -13,15 +13,42 @@ import { get } from 'lodash';
 import AutoDirection from 'components/auto-direction';
 import { getCurrentUser } from 'state/current-user/selectors';
 
+const TEXTAREA_MAX_HEIGHT = 236; // 10 lines
+const TEXTAREA_MIN_HEIGHT_COLLAPSED = 47;
+const TEXTAREA_MIN_HEIGHT_FOCUSED = 68; // 2 lines
+const TEXTAREA_VERTICAL_BORDER = 2;
+
 export class CommentDetailReply extends Component {
 	state = {
 		commentText: '',
 		hasFocus: false,
+		textareaMinHeight: TEXTAREA_MIN_HEIGHT_COLLAPSED,
 	};
 
-	handleTextChange = event => this.setState( { commentText: event.target.value } );
+	bindTextareaRef = textarea => {
+		this.textarea = textarea;
+	}
+
+	calculateTextareaMinHeight = () => {
+		const textareaScrollHeight = this.textarea.scrollHeight;
+		const textareaMinHeight = Math.min( TEXTAREA_MAX_HEIGHT, textareaScrollHeight + TEXTAREA_VERTICAL_BORDER );
```

Comment:
Is it possible to set a fixed min-height, max-height, and then let the js update the height attribute?